PSKoans/Public/Measure-Karma.ps1

```diff
@@ -4,7 +4,7 @@
         Reflect on your progress and check your answers.
     .DESCRIPTION
         Get-Enlightenment executes Pester against the koans to evaluate if you have made the necessary
-        corrections for success.
+        corrections for success. If you'd like to use VS Code Insiders, set $env:PSKoans_EditorPreference equal to "code-insiders".
```

Comment:
Please insert a line break after the variable name here. I'm not a fan of long comment-help lines. 😄